R/plot_taxa.R

```diff
@@ -1296,7 +1296,7 @@ plot_sites <- function(
   validate_arguments(fun.name = "plot", fun.args = as.list(environment()))
 
   # check for suggested packages that are required for this function to work
-  suggs <- c("ggrepel", "usmap","maps")
+  suggs <- c("ggrepel", "usmap","maps","sf")
```

Comment:
@sokole @RafaelRangel0 The "maps" package is not listed in the package DESCRIPTION file. Should it be?

Reply:
I can add this in